include/fmt/base.h

```diff
@@ -1484,6 +1492,20 @@ template <typename Context> struct arg_mapper {
   FMT_MAP_API auto map(double val) -> double { return val; }
   FMT_MAP_API auto map(long double val) -> long double { return val; }
 
+#if FMT_USE_BITINT
+#  pragma clang diagnostic push
+#  pragma clang diagnostic ignored "-Wbit-int-extension"
+  template <class T, int N = 0> struct is_bitint : std::false_type {};
```

Comment:
This should be defined outside of conditional block so that the check could be used on all platforms.